samples/event-sourcing-esdb-simple/src/test/java/io/eventdriven/ecommerce/api/ECommerceApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package io.eventdriven.ecommerce.api;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class ECommerceApplicationTests {
```

Comment:
No tests :O !?
Shame of you! :D 

I propose to get faimilar with Spock framework for tests.
Some examples:
https://github.com/ddd-by-examples/library/blob/master/src/test/groovy/io/pillopl/library/lending/book/application/DuplicateHoldFoundTest.groovy

You may also test your architecture constraints with ArchUnit.
https://www.archunit.org/

Reply:
As you've already seen, I added a set of integration tests :)